apps/s_server.c

```diff
@@ -469,14 +469,14 @@ static tlsextstatusctx tlscstatp = { -1 };
  * the OCSP certificate IDs and minimise the number of OCSP responses by caching
  * them until they were considered "expired".
  */
-static int get_ocsp_resp_from_responder(SSL *s, tlsextstatusctx *srctx,
-                                        OCSP_RESPONSE **resp)
+static int get_ocsp_resp_from_responder_single(SSL *s, X509 *x, 
```

Comment:
the added argument `X509 *x,` does not seems to be used, no ?

Reply:
the certificate x is used to get the AIA from it (line 489).

Reply:
ok, seen